ksqldb-engine-common/src/main/java/io/confluent/ksql/schema/ksql/SqlTimestamps.java

```diff
@@ -17,12 +17,14 @@
 
 import io.confluent.ksql.util.KsqlConstants;
 import io.confluent.ksql.util.timestamp.PartialStringToTimestampParser;
+import java.sql.Time;
 import java.sql.Timestamp;
+import java.time.LocalTime;
 import java.time.ZoneId;
 import java.time.format.DateTimeFormatter;
 
 /**
- * Helpers for working with Sql {@code TIMESTAMP}.
+ * Helpers for working with SQL time types.
  */
 public final class SqlTimestamps {
```

Comment:
If this also work with other time types, should we rename the class to reflect that?

Reply:
Yeah. I was planning to do this in another PR, so that this one would not have so many files (this class gets referenced several times throughout the codebase)